src/app/students/browsecredits/CreditList.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+
+import CreditsData from "./credits.json";
+import Image from "next/image";
+import Link from "next/link";
+
+type Credit = {
+	id: string;
+	url: string;
+	creditname: string;
+};
+
+type Credits = Credit[];
+
+type SearchValueProps = {
+	search: string;
+};
+
+const CreditList = ({ search }: SearchValueProps) => {
+	const Credits: Credits = CreditsData;
+
+	return (
+		<>
+			{Credits && (
+				<>
+					{Credits.filter((credit) => {
+						return search.toLowerCase() === ""
+							? credit
+							: credit.creditname.toLowerCase().includes(search);
+					}).map((credit) => (
+						<div key={credit.id}>
+							<figure className="h-[12.5rem] w-[12.5rem] md:h-[7.8125rem] md:w-[7.8125rem] lg:h-[12.5rem] lg:w-[12.5rem] ">
```

Comment:
why is the set width and height? what is this for?

Reply:
Thanks for catching this, removed the width and height.